<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashWidget</class>
 <widget class="QWidget" name="SplashWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SplashWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background: white;
}</string>
  </property>
  <widget class="QLabel" name="label_msg">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>560</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Initializing...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_copyright">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>410</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>@ Copyright 2020 FutureDial Inc.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_version">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>10</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>v1.0.0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
